docs/end-user-topics/dois.rst

```diff
@@ -0,0 +1,89 @@
+DOIs
+====
+
+Introduction
+------------
+A Digial Object Identifier (DOI) is a permanent identifier that can be used in publications to identify the exact
+version of a workflow or tool (entry). DOIs are commonly referenced from academic papers.
+See the `DOI Foundation <https://www.doi.org>`__ for more details.
+
+Creating and assigning DOIs is commonly known as "minting". Many repositories mint DOIs. Dockstore currently
+only supports `Zenodo <https://zenodo.org>`__ DOIs. Dockstore uses Zenodo to mint DOIs, and it can also recognize DOIs generated by Zenodo outside of Dockstore.
+
+Zenodo supports `DOI versioning <https://support.zenodo.org/help/en-gb/1-upload-deposit/97-what-is-doi-versioning>`__  with a concept DOI for an overall record, and version-specific DOIs.
+
+In Dockstore, the concept DOI is associated with the entry as a whole, and version DOIs are associated with entry versions.
+
+Dockstore Support for DOIs
+--------------------------
+
+Dockstore can generate DOIs for entries, as well as recognize some DOIs generated outside of Dockstore. Dockstore displays DOIs associated with entries. If you have multiple concept DOIs associated
+with a single entry, you can specify which DOI to display.
+
+Dockstore generates DOIs for entries 2 ways:
+
+* Explicitly initiated by a Dockstore user. This approach requires you linking your Zenodo account. You select a version via the Dockstore UI and click to create a DOI. See :doc:`Creating Snapshots & Requesting DOIs </advanced-topics/snapshot-and-doi>` for more details
+* Dockstore can automatically generate DOIs for your entries without you needing to link your Zenodo account. These DOIs are created in the Dockstore Zenodo Community
+
+In addition, Dockstore can discover DOIs created by a Zenodo-GitHub integration.
+
+Choosing which DOI to display
+-----------------------------
+
+When viewing the public page of an entry, the entry's concept DOI and the selected version's DOI are displayed.
+
+.. figure:: /assets/images/docs/show-doi.png
+   :alt: Show DOIs
+
+Dockstore only displays one DOI associated with a entry, even if there is more than one DOI, e.g., one generated by GitHub-Zenodo integration, and another generated by Dockstore.
+If this situation arises, you can control which DOI is displayed via your dashboard. Select the entry, then click "Manage DOIs", and you'll get a dialog like this. Select your choice
+and click "Save".
+
+   
+.. figure:: /assets/images/docs/manage-dois.png
+   :alt: Manage DOIs
+
+
+Features of the different ways of issuing DOIs
```

Comment:
Don't feel super strongly about it, but suggesting we use Overview instead of Features